components/content/Callout.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="bg-surface-secondary rounded-lg">
+    <div class="flex p-5 space-x-5">
+      <information-circle class="icon-md text-static-black" />
+      <div class="flex flex-col align space-y-1 text-sm leading-5">
+        <div
+          v-if="heading"
+          class="text-content-secondary font-medium"
+        >
+          {{ heading }}
+        </div>
+        <p class="text-content-tertiary m-0 font-normal">
+          {{ description }}
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+const props = defineProps({
+  heading: { type: String, required: false, default: '' },
```

Comment:
@AndyClifford FYI we don't actually have a requirement for a heading at this stage